dev-tools/build_release.py

```diff
@@ -618,6 +618,47 @@ def check_norelease(path='src'):
           if pattern.search(line):
             raise RuntimeError('Found //norelease comment in %s line %s' % (full_path, line_number))
 
+def run_and_print(text, run_function):
+  try:
+    print(text, end='')
+    run_function()
+    print(OKGREEN + 'OK' + ENDC)
+  except RuntimeError:
+    print(FAIL + 'NOT OK' + ENDC)
+
+def check_env_var(text, env_var):
+  try:
+    print(text, end='')
+    env[env_var]
+    print(OKGREEN + 'OK' + ENDC)
+  except KeyError:
+    print(FAIL + 'NOT OK' + ENDC)
+
+def check_environment_and_commandline_tools():
+  check_env_var('Checking for AWS env configuration AWS_SECRET_ACCESS_KEY_ID...     ', 'AWS_SECRET_ACCESS_KEY')
+  check_env_var('Checking for AWS env configuration AWS_ACCESS_KEY_ID...            ', 'AWS_ACCESS_KEY_ID')
+  check_env_var('Checking for SONATYPE env configuration SONATYPE_USERNAME...       ', 'SONATYPE_USERNAME')
```

Comment:
This variable is not used in this script. Is this check done for another script? May be we could add a comment at the beginning of this script?